src/index.html

```diff
@@ -1,13 +1,283 @@
+<!--#region-->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet"> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;700;800&display=swap" rel="stylesheet">
+    <link rel="icon" href="images/favicon.png">
+    <title>MyBike</title>
+    <link rel="stylesheet" href="styles/main.scss">
+    <script src="scripts/main.js"></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!--#endregion-->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <div class="top-bar__block">
+              <a href="#" class="top-bar__logo-link">
+                <img src="images/MyBike-white.svg"
```

Comment:
```suggestion
                <img 
                  src="images/MyBike-white.svg"
```
Consider fixing indentations accordingly to the code style guide